README.md

```diff
@@ -25,6 +25,10 @@ The location at which the `jenkins-cli.jar` jarfile will be kept. This is used f
       - sonar
       - ssh
 
+By default jenkins installs all your plugins but does not update any of the currently installed plugins. If you would like the currently installed plugins updated, set jenkins_plugin_update to true
```

Comment:
This description needs to go under the variable, and the whole addition needs to move under the block of text that follows it (my readmes go variable, then description).